### PR TITLE
release 2.12.0 version updates

### PR DESCRIPTION
**Pull request by mozrokafor**
release 2.12.0 version updates

### CODE REVIEW

**Review by birdsarah — CHANGES_REQUESTED** [review]
At this stage of the release process you should be updated the staging info not production.

File: docs/mozillavpn_stage.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "latest": {
-    "ANDROID": "2.10.1",
-    "IOS": "2.10.0",
-    "LINUX": "2.11.0",
-    "MACOS": "2.11.0",
-    "WINDOWS": "2.11.0"
+    "ANDROID": "2.12.0",
+    "IOS": "2.12.0",
```
Comment (bakulf) [inline docs/mozillavpn_stage.json:4]: iOS is still 2.11 until the review is completed

File: docs/mozillavpn_stage.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "latest": {
-    "ANDROID": "2.10.1",
-    "IOS": "2.10.0",
-    "LINUX": "2.11.0",
-    "MACOS": "2.11.0",
-    "WINDOWS": "2.11.0"
+    "ANDROID": "2.12.0",
```
Comment (bakulf) [inline docs/mozillavpn_stage.json:3]: Android is still 2.11 until the review is completed

File: docs/mozillavpn_stage.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+    "IOS": "2.12.0",
+    "LINUX": "2.12.0",
```
Comment (bakulf) [inline docs/mozillavpn_stage.json:5]: is linux 2.12 released?

File: docs/mozillavpn_stage.json [hunk 4]
```diff
@@ -78,6 +78,11 @@
       "build": null,
       "date": "2022-11-02",
       "platforms": ["ANDROID", "IOS", "LINUX", "MACOS", "WINDOWS"]
+    },
+    "2.12.0": {
+      "build": null,
+      "date": "2022-12-06",
+      "platforms": ["ANDROID", "IOS", "LINUX", "MACOS", "WINDOWS"]
```
Comment (bakulf) [inline docs/mozillavpn_stage.json:85]: remove LINUX, ANDROID, IOS if they are not ready.

**Review by bakulf — CHANGES_REQUESTED** [review]

File: docs/mozillavpn_stage.json [hunk 5]
```diff
@@ -78,6 +78,11 @@
       "build": null,
       "date": "2022-11-02",
       "platforms": ["ANDROID", "IOS", "LINUX", "MACOS", "WINDOWS"]
+    },
+    "2.12.0": {
+      "build": null,
+      "date": "2022-12-06",
```
Comment (bakulf) [inline docs/mozillavpn_stage.json:84]: why this date?

**Review by bakulf — CHANGES_REQUESTED** [review]

File: docs/mozillavpn.json [hunk 6]
```diff
@@ -3,8 +3,8 @@
     "ANDROID": "2.11.0",
     "IOS": "2.11.1",
     "LINUX": "2.11.0",
-    "MACOS": "2.12.0",
```
Comment (birdsarah) [inline docs/mozillavpn.json:6]: why is this PR attempting to revert the release of 2.12?

**Review by birdsarah — CHANGES_REQUESTED** [review]
I don't understand this PR.

Please add some explanatory text or close it.

**Comment by mozrokafor** [comment]
https://github.com/mozilla/mozillavpn-product-details/pull/53 migrating to

**Pull request closed without merging**